test/extended/util/test.go

```diff
@@ -139,6 +142,14 @@ func verifyTestSuitePreconditions() {
 		// uid don't get forced into a range (mimics upstream
 		// behavior)
 		addE2EServiceAccountsToSCC(c, namespaces, "anyuid")
+
+		// The intra-pod test requires that the service account have
+		// permission to retrieve service endpoints.
+		osClient, _, err := configapi.GetOpenShiftClient(KubeConfigPath())
+		if err != nil {
+			FatalErr(err)
+		}
+		addRoleToE2EServiceAccounts(osClient, namespaces, bootstrappolicy.EditRoleName)
```

Comment:
@smarterclayton and so it begins. I can't decide if I want to just make all the upstream service accounts cluster admins and stop worrying about it, or actually bound their permissions and at least know when the upstream e2e's are bumping the limits